test/fixtures/ruby_gems_users.yml

```diff
@@ -0,0 +1,16 @@
+
+one:
+  user_id: 1
+  ruby_gem_id: 1
+
+two:
+  user_id: 1
+  ruby_gem_id: 2
+
+three:
+  user_id: 1
+  ruby_gem_id: 3
+
+four:
+  user_id: 3
+  ruby_gem_id: 4
```

Comment:
What's the purpose of these fixtures? I don't think we have a `RubyGemsUser` model.